lib/vfscore/rootfs.c

```diff
@@ -40,6 +42,11 @@
 #include <sys/stat.h>
 #include <sys/mount.h>
 #include <uk/init.h>
+#ifdef CONFIG_LIBINITRAMFS
+#include <uk/plat/memory.h>
+#include <uk/cpio.h>
+#include <string.h>
+#endif
```

Comment:
In general I think good practice is to include the config option to the endif part. This makes the macro conditions sometimes more readable - especially if there are long blocks:
``` c
#endif /* CONFIG_LIBVFSCORE_ROOTFS_INITRD */
```